Equilobe.DailyReport.Models/ReportFrame/JiraAuthor.cs

```diff
@@ -78,9 +82,16 @@ public string RemainingChartPixelWidthString
         public bool HasAssignedIssues { get; set; }
         public bool HasDayLogs { get; set; }
         public int MaxHourValue { get; set; }
+        public bool HasSprint { get; set; }
 
         public SummaryGuidelineInfo GuidelineInfo { get; set; }
 
+        public UncompletedTasks InProgressTasksModel { get; set; }
+        public UncompletedTasks OpenTasksModel { get; set; }
+        public UncompletedTasks UncompletedTasksModel { get; set; }
```

Comment:
Rename without Model and keep only these properties and remove the separate collections of IssueDetailed.